CMakeLists.txt

```diff
@@ -5,10 +5,20 @@
 cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
 project(VAST C CXX)
 
+option(VAST_RELOCATEABLE_INSTALL "Make the installation relocateable")
+
 # Silence warning CMP0042
 if (APPLE AND NOT DEFINED CMAKE_MACOSX_RPATH)
-  set(CMAKE_MACOSX_RPATH true)
-endif()
+  set(CMAKE_MACOSX_RPATH TRUE)
+  set(CMAKE_SKIP_BUILD_RPATH  FALSE)
+  set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+
```

Comment:
Removing this extra newline is the only thing that I'd have to say about this PR. Otherwise looks good!